ngraph/test/backend_api.cpp

```diff
@@ -58,35 +58,3 @@ TEST(backend_api, DISABLED_config_unsupported)
     EXPECT_FALSE(backend->set_config(config, error));
     EXPECT_FALSE(error == "");
 }
-
-#ifndef NGRAPH_JSON_DISABLE
-TEST(backend_api, save_load)
-{
-    Shape shape{2, 2};
-    auto A = make_shared<op::Parameter>(element::f32, shape);
-    auto B = make_shared<op::Parameter>(element::f32, shape);
-    auto f = make_shared<Function>(make_shared<op::Add>(A, B), ParameterVector{A, B});
```

Comment:
I think you should rather change the op in this test to a different op rather than removing it. It's not the addition operation itself itested here, it checks the save/load interface of a backend.

Reply:
Ok, I will change it. Thank you!